config/deprecation-workflow.js

```diff
@@ -2,23 +2,25 @@
 window.deprecationWorkflow = window.deprecationWorkflow || {};
 window.deprecationWorkflow.config = {
   workflow: [
-    { handler: "silence", matchId: "ember-global" },
-    { handler: "silence", matchId: "ember.component.reopen" },
-    { handler: "silence", matchId: "ember-metal.get-with-default" },
+    { handler: "throw", matchId: "ember-global" },
+    { handler: "silence", matchId: "ember.component.reopen" }, // not finish yet
+    { handler: "throw", matchId: "ember-metal.get-with-default" },
     { handler: "silence", matchId: "implicit-injections" },
     {
-      handler: "silence",
+      // Fixed
+      handler: "throw",
       matchId: "ember-views.curly-components.jquery-element",
     },
```

Comment:
You can delete all *fixed* issue like proposed in the deprecation [workflow](https://github.com/mixonic/ember-cli-deprecation-workflow#workflow). But be sure to test the app once in a smoke test while having them turned on (*throw*).